proto/zitadel/admin.proto

```diff
@@ -4039,6 +4040,9 @@ message DataOrg {
 
     repeated zitadel.idp.v1.IDPUserLink user_links = 36;
     repeated zitadel.org.v1.Domain domains = 37;
+
+    repeated DataAppKey app_keys = 38;
+    repeated DataMachineKey machine_keys = 39;
```

Comment:
no reason, can be added to the messages

Reply:
IMO it would be more logical to have a single object containing all parts (e.g. application as a whole with all its settings, which includes keys)... even if the API is like that, the export/import is a different use case and in my mind it feels more natural to import the whole application
but as the current export/import message is already structured like this (multiple submessages), i'm ok with it